pandas/tests/frame/conftest.py

```diff
@@ -127,6 +127,22 @@ def timezone_frame():
     return df
 
 
+@pytest.fixture
+def datetime_series():
```

Comment:
don't add any more here

Reply:
not here for now

Reply:
@jreback 
See above:
> PS. As noted in the other threads, this is a TestData-"fixture" that's used in several modules.

This is used also used in #25627, for example

Reply:
i don't think these fixtures useful; if you really really want them put them in the actual test file for now